frontendGraphTableProvider/src/main/java/com/virtuslab/gitmachete/frontend/ui/api/table/VcsRootComboBox.java

```diff
@@ -89,7 +89,7 @@ private void updateRepositories() {
   }
 
   @Override
-  @UIEffect
+  @SuppressWarnings("guieffect:call.invalid.ui") // Actually this is ui safe
```

Comment:
Why is that suppression needed? B/c of `getModel()`? Is `getModel()` UI-safe?

Reply:
If so, consider stubbing `getModel` as `@SafeEffect`; if not, refine the comment here to explain why we consider this specific invocation of `getModel` to be UI-safe